--- lightdm-1.9.11/data/lightdm.conf.fedora_config	2014-03-12 14:56:34.000000000 -0500
+++ lightdm-1.9.11/data/lightdm.conf	2014-03-13 07:15:20.266046171 -0500
@@ -19,9 +19,9 @@
 #start-default-seat=true
 #greeter-user=lightdm
 #minimum-display-number=0
-#minimum-vt=7
+minimum-vt=1
 #lock-memory=true
-#user-authority-in-system-dir=false
+user-authority-in-system-dir=true
 #guest-account-script=guest-account
 #log-directory=/var/log/lightdm
 #run-directory=/var/run/lightdm
@@ -73,7 +73,7 @@
 [SeatDefaults]
 #type=xlocal
 #xdg-seat=seat0
-#xserver-command=X
+xserver-command=X -background none
 #xserver-layout=
 #xserver-config=
 #xserver-allow-tcp=false
@@ -85,7 +85,7 @@
 #xdmcp-key=
 #unity-compositor-command=unity-system-compositor
 #unity-compositor-timeout=60
-#greeter-session=example-gtk-gnome
+greeter-session=lightdm-greeter
 #greeter-hide-users=false
 #greeter-allow-guest=true
 #greeter-show-manual-login=false
@@ -93,7 +93,7 @@
 #user-session=default
 #allow-guest=true
 #guest-session=
-#session-wrapper=lightdm-session
+session-wrapper=/etc/X11/xinit/Xsession
 #greeter-wrapper=
 #guest-wrapper=
 #display-setup-script=
diff -up lightdm-1.9.11/data/users.conf.fedora_config lightdm-1.9.11/data/users.conf
--- lightdm-1.9.11/data/users.conf.fedora_config	2013-07-30 09:09:28.000000000 -0500
+++ lightdm-1.9.11/data/users.conf	2014-03-13 07:13:28.320086064 -0500
@@ -9,6 +9,6 @@
 # hidden-shells = Shells that indicate a user cannot login
 #
 [UserAccounts]
-minimum-uid=500
+minimum-uid=1000
 hidden-users=nobody nobody4 noaccess
 hidden-shells=/bin/false /usr/sbin/nologin